src/components/SkillRatingCard/SkillRatingCard.js

```diff
@@ -1,5 +1,10 @@
 // Packages
```

Comment:
Keep this section for Core React modules only.

Reply:
Core react module? You mean just `React, { Component }` ?

Reply:
Also, `propTypes`, since it is common to all the React components.